ts/components/leftpane/overlay/OverlayClosedGroup.tsx

```diff
@@ -94,12 +94,12 @@ export const OverlayClosedGroupV2 = () => {
     if (selectedMemberIds.length < 1) {
       ToastUtils.pushToastError(
         'pickClosedGroupMember',
-        localize('groupCreateErrorNoMembers').toString()
+        tr('groupCreateErrorNoMembers').toString()
       );
       return;
     }
     if (selectedMemberIds.length >= VALIDATION.CLOSED_GROUP_SIZE_LIMIT) {
-      ToastUtils.pushToastError('closedGroupMaxSize', localize('groupAddMemberMaximum').toString());
+      ToastUtils.pushToastError('closedGroupMaxSize', tr('groupAddMemberMaximum'));
```

Comment:
Strip like the other toasts? IIRC we strip all to toasts in because toasts render HTML? I don't quite remember but I saw you did this with other toast calls